### PR TITLE
Edit README syntax to make syntax readable.

### PR DESCRIPTION
**Pull request by rosiel**
When discussing how to escape characters, the escape characters are getting swallowed up in markdown syntax. 

# Interested parties
@willtp87 @Islandora-Labs/committers

### CODE REVIEW

**Review by DonRichards — APPROVED** [review]
That looks good, thanks for the added clarification! I'll wait for a maintainer to merge.

**Comment by DiegoPino** [comment]
@DonRichards feel free to merge. The CONTRIBUTING.MD file allows same rules as other Repos. So go for it!

**Pull request merged**
